rotkehlchen/constants/ethereum.py

```diff
@@ -111,6 +113,8 @@ def abi(name: str) -> List[Dict[str, Any]]:
 MAKERDAO_UNI_A_JOIN = EthereumConstants().contract('MAKERDAO_UNI_A_JOIN')
 MAKERDAO_RENBTC_A_JOIN = EthereumConstants().contract('MAKERDAO_RENBTC_A_JOIN')
 MAKERDAO_AAVE_A_JOIN = EthereumConstants().contract('MAKERDAO_AAVE_A_JOIN')
+MAKERDAO_WSTETH_A_JOIN = EthereumConstants().contract('MAKERDAO_WSTETH_A_JOIN')
```

Comment:
You seem to be adding this in the constants without actually adding support for this. Please either also support it and add to changelog or completely remove from here and from contracts json.

Reply:
Was there for later but okey, removing it